dist/spec.json

```diff
@@ -13,7 +13,7 @@
       "name": "Apache-2.0",
       "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
     },
-    "version": "2.2.1"
+    "version": "2.3.0"
```

Comment:
@bretterer this should be 2.2.5 instead, right? (considering latest is 2.2.4, presently)

Reply:
You are targeting the master branch?  If so, this will be a 2.3.0 release

If you are trying to do a bug fix, please update the target here to be the 2.2 branch

Reply:
I'll consider this a bug fix, so will change the target to 2.2 branch.